testng-core/src/test/kotlin/org/testng/internal/samples/classhelper/issue1339/GrandpaBearSample.kt

```diff
@@ -0,0 +1,9 @@
+package org.testng.internal.samples.classhelper.issue1339;
+
+open class GrandpaBearSample {
```

Comment:
Is it working if sample classes are not open?

Reply:
I didnt understand this comment. The class cannot be extended if its not open and there are other samples which test the regular use case.

Reply:
You right "not" open means "final".
What about `sealed`?

The question behind: what is the best keyword?

Reply:
sealed doesn't exist in java (not yet)
maybe sealed is better than public

it's a global kotlin question because we don't care for tests.